src/Solvers/batched_tridiagonal_solver.jl

```diff
@@ -73,38 +70,123 @@ Reference implementation per Numerical Recipes, Press et. al 1992 (§ 2.4).
 """
 function solve!(ϕ, solver::BatchedTridiagonalSolver, rhs, args... )
 
-    a, b, c, t, parameters = solver.a, solver.b, solver.c, solver.t, solver.parameters
-    grid = solver.grid
-
-    launch!(architecture(solver), grid, :xy,
-            solve_batched_tridiagonal_system_kernel!, ϕ, a, b, c, rhs, t, grid, parameters, Tuple(args))
+    launch_config = if solver.tridiagonal_direction == :x
+                        :yz
+                    elseif solver.tridiagonal_direction == :y
+                        :xz
+                    elseif solver.tridiagonal_direction == :z
```

Comment:
```suggestion
                    elseif solver.tridiagonal_direction isa ZDirection
```

this is what you change

Reply:
Yes, thanks for the help!

I actually did the right thing, but for some reason (I think Revise having some trouble) things weren't compiling properly. I restarted the REPL and things now seem to be working.